ftest/web-ui/webdriver/test/features/comments.feature

```diff
@@ -0,0 +1,102 @@
+@watch
+Feature: Comments
+
+  Background:
+    Given user "John" exists in group "members"
+    And user "Susan" exists in group "administrators"
+    And I have a File document
+
+  Scenario: I can see document comment thread with multiple comments
+    Given I have the following comment thread
+      | author | text      |
+      | John   | art       |
+      | Susan  | books     |
+      | John   | cats      |
+      | John   | dogs      |
+      | Susan  | expensive |
+      | John   | failure   |
+      | John   | general   |
+      | Susan  | hardware  |
+      | John   | image     |
+      | John   | java      |
+      | John   | kickoff   |
+      | John   | languages |
+    And John comment "languages" has the following replies
+      | author | text       |
+      | Susan  | portuguese |
+      | John   | spanish    |
+      | Susan  | french     |
+      | John   | german     |
+      | Susan  | english    |
+      | John   | italian    |
+      | Susan  | russian    |
+      | Susan  | polish     |
+      | Susan  | arabic     |
+      | Susan  | dutch      |
+      | John   | croatian   |
+    And I login as "John"
+    When I browse to the document
+    Then I can see the comment thread has 10 visible items
+    And I can see the comment thread has a total of 12 items to be loaded
+    When I load all comments
+    Then I can see the comment thread has 12 visible items
+    And I can see my comment "languages" has a reply thread with 11 replies
```

Comment:
I can see my comment "languages" has 11 replies ?

Reply:
Since there is a similar step "... has X visible replies", I tried to have this a little bit different to represent the collapsed thread.